Makefile

```diff
@@ -0,0 +1,99 @@
+K8S_DIR := .k8s
+K8S_CLUSTER_NAME := storj
+KUBECONFIG = $(realpath $(K8S_DIR)/config)
+BUCKET_NAME ?= velero
+STORJ_ACCESS ?= # Required
+DOCKER_IMAGE := storjthirdparty/velero-plugin
+DOCKER_TAG ?= dev
+VELERO_BACKUP_LOCATION := default
+
+export KUBECONFIG
+
+# check minimun required tools
+$(if $(shell which kind),,$(error kind tool is required))
+$(if $(shell which velero),,$(error velero tool is required))
+$(if $(shell which kubectl),,$(error kubectl tool is required))
+
+define HELP_MSG
+Run a reproducible local development environment to develop the Storj Velero \
+plugin. Usage: make [target]
+endef
+
+.PHONY: help
+help: ## show this help message
+	@echo $(HELP_MSG)
+	@cat Makefile | awk -F ":.*##"  '/##/ { printf "    %-30s %s\n", $$1, $$2 }' | grep -v  grep
+
+.PHONY: k8s-start
+k8s-start: ## start and initialize a local K8s cluster
+	@if [ ! -f $(K8S_DIR)/config ]; then \
+		mkdir -p $(K8S_DIR); \
+		kind create cluster --kubeconfig $(K8S_DIR)/config --name $(K8S_CLUSTER_NAME) --wait 1m; \
+	fi
+
+.PHONY: velero-install
+velero-install: k8s-start ## install Velero and this plugin into the K8s cluster
+	@velero install --no-secret \
+		--no-default-backup-location \
+		--use-volume-snapshots=false \
+		--wait
+
+.PHONY: velero-plugin-add
+velero-plugin-add: ## add this plugin to Velero server
+	@velero plugin add $(DOCKER_IMAGE):$(DOCKER_TAG)
+
+.PHONY: velero-plugin-remove
+velero-plugin-remove: ## delete this plugin from Velero server
+	@velero plugin remove $(DOCKER_IMAGE):$(DOCKER_TAG)
+
+.PHONY: velero-backup-location-create
+velero-backup-location-create: .is-access-set ## create a backup location using Storj plugin
+	@velero backup-location create $(VELERO_BACKUP_LOCATION) \
+		--bucket=$(BUCKET_NAME) \
+		--config accessGrant=$(STORJ_ACCESS) \
+		--provider=gcp
+
+.PHONY: velero-backup-location-delete
+velero-backup-location-delete: ## delete the backup location that uses Storj plugin
+		@kubectl -n velero delete backupstoragelocation.velero.io $(VELERO_BACKUP_LOCATION)
+
+.PHONY: velero-uninstall
+velero-uninstall: ## uninstall Velero from the K8s cluster
+ifeq ($(call is_k8s_running),)
+	@kubectl delete namespace/velero clusterrolebinding/velero
+	@kubectl delete crds -l component=velero
+endif
+
+.PHONY: plugin-build
+plugin-build: ## build docker image of this Velero plugin for development
+	@docker image build --squash --rm -t $(DOCKER_IMAGE):$(DOCKER_TAG) .
```

Comment:
`--squash` seems to be supported only in docker experimental mode

Reply:
oh, I didn't know, I will remove it.